html/src/TDocOutput.cxx

```diff
@@ -359,10 +359,10 @@ void TDocOutput::Convert(std::istream& in, const char* infilename,
                 && outfile_stat.fMtime > infile_stat.fMtime) {
                // the first canvas file exists and it is newer than the script, so we reuse
                // the canvas files. We need to know how many there are:
-               dirHandle = gSystem->OpenDirectory(gSystem->DirName(htmlFilename));
+               dirHandle = gSystem->OpenDirectory(gSystem->GetDirName(htmlFilename));
                TString stem(gSystem->BaseName(outfilename));
                stem += "_";
-               TString dir(gSystem->DirName(htmlFilename));
+               TString dir = gSystem->GetDirName(htmlFilename);
```

Comment:
Isn't
```suggestion
               TString dir(gSystem->GetDirName(htmlFilename));
```
slightly more efficient (allow optimizing away one of the TString construction)?

Reply:
No.